pkg/dmi.go

```diff
@@ -0,0 +1,169 @@
+package pkg
+
+import (
+	"bufio"
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+type Entry = map[string][]string
+type DMIMap = map[string]Entry
+
+type DMIDecoder struct {
+	installed  bool
```

Comment:
for separation of concertns, the DMIDecoder should only parse data, into an internal map, it shouldn't know about external software installed e.g `sudo, dmidecode`, it should be a concern of another function, or the user provides it